rioxarray/_io.py

```diff
@@ -1263,6 +1263,12 @@ def open_rasterio(
     if chunks is not None:
         result = _prepare_dask(result, riods, filename, chunks)
 
+    result.encoding["preferred_chunks"] = {
+        "y": riods.block_shapes[0][0],
```

Comment:
```suggestion
        riods.rio.y_dim: riods.block_shapes[0][0],
```

Reply:
hmm, this doesn't seem do be defined here...